src/Deposit.js

```diff
@@ -438,6 +438,10 @@ export default class Deposit {
     return toBN(await this.contract.methods.lotSizeSatoshis().call())
   }
 
+  async getTBTCLotSize() {
+    return toBN(await this.contract.methods.lotSizeTbtc().call())
+  }
```

Comment:
… This might be the right moment to rename these to match what we're using on the contract side. No idea why I didn't do this in the first place 🤔

Reply:
Yeaaah, that'd be nice.